wicamo96/Verti-Comps
# Styling

## What Changed

- Added consistent styling across all pages and users
- Tweaked wording on some of the pages for clarity and fixed typos

## How To Test

- Login as each user type and view general styling
- All styling should be consistent, no stand out differences on any individual page should exist
